frontend/www/js/omegaup/components/user/Profilev2.vue

```diff
@@ -0,0 +1,289 @@
+<template>
+  <div class="container-fluid p-0 mt-0" data-user-profile-root>
+    <div class="row">
+      <div class="col-md-2">
+        <div class="card">
+          <omegaup-countryflag
+            class="m-1"
+            v-bind:country="profile.country_id"
+            v-if="profile.country_id"
+          />
+          <div class="card-body">
+            <div class="img-thumbnail rounded-circle bottom-margin">
+              <img class="rounded-circle" v-bind:src="profile.gravatar_92" />
+            </div>
+          </div>
+          <div class="card-title text-center">
+            <div class="mb-3">
+              <omegaup-user-username
+                v-bind:classname="profile.classname"
+                v-bind:username="profile.username"
+              ></omegaup-user-username>
+            </div>
+            <div class="mb-3">
+              <h4 class="m-0">
+                {{
+                  profile.rankinfo.rank > 0 ? `#${profile.rankinfo.rank}` : ''
+                }}
+              </h4>
+              <p>
+                <small>
+                  {{ T.wordsRanking }}
+                </small>
+              </p>
+            </div>
+            <div class="mb-3">
+              <h4 class="m-0">
+                {{ Object.keys(solvedProblems).length }}
+              </h4>
+              <p>
+                <small>{{ T.profileSolvedProblems }}</small>
+              </p>
+            </div>
+            <div class="mb-3">
+              <h5 class="m-0">
+                {{
+                  profile.programming_languages[
+                    profile.preferred_language
+                  ].split(' ')[0]
+                }}
+              </h5>
+              <p>
+                <small>{{ T.wordsProgrammingLanguage }}</small>
```

Comment:
```suggestion
                <small>{{ T.userEditPreferredProgrammingLanguage }}</small>
```

esto ya está definido! si quieres puedes cambiar el nombre a `profilePreferredProgrammingLanguage` en el archivo de cadenas de traducción.